### PR TITLE
Return promise or use callback, closes #1, fixes #3; Release v0.2.0

### LINKED ISSUE #1: return promise
I suggest returning a [promise](https://github.com/petkaantonov/bluebird) from `snowsh.post` to allow for some nice async behaviors. this wouldn't be too difficult to implement either since the lib only has two behaviors

### LINKED ISSUE #3: Mistake in require OAuth statement
In snowshoe.js

 `var OAuth = require('OAuth');`

needs to be

`var OAuth = require('oauth');`

First one seems to work on my local machine (Mac) but fails on Heroku.

### PR DESCRIPTION
**Pull request by RasPhilCo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +37,29 @@ client.post(data, response, function(error, data, response){
...
+})
+```
+
+Or return a promise instead (via Q).
+
+```javascript
+client.post(data).then(function (data) {
+    // handle success
+  })
+  .fail(function (error) {
```
Comment (kylemac) [inline README.md]: you may want to use `.catch()` instead of `.fail()` – i think `fail` is not part of the spec (any longer).

File: README.md [hunk 2]
```diff
@@ -38,6 +37,29 @@ client.post(data, response, function(error, data, response){
...
+    // handle success
```
Comment (kylemac) [inline README.md]: i think you mean, `handle errors`

**Comment by kylemac** [comment]
minor comments, the rest looks :thu

**Comment by RasPhilCo** [comment]
Edited. Thanks, @kylemac.

**Pull request merged**